docs/Users/Install.rst

```diff
@@ -158,13 +158,20 @@ https://docs.python.org/3.3/install/#alternate-installation.
     If you are using a proxy, follow these steps:
 
     -  Set up your system-wide proxy.
-    -  Use ``sudo -E pip3 install coala`` (the ``-E`` flag takes the
-       existing environment variables into the ``sudo`` environment).
+    -  Use
 
-    You could also set your pip.conf file to use a proxy, to know more
-    read
-    http://stackoverflow.com/questions/14149422/using-pip-behind-a-proxy
-    for further clarification.
+    ::
```

Comment:
This shouldnt be modified in this commit which is supposed to only add the `$` thing